src/test/java/com/google/cloud/solutions/datalineage/extractor/BigQueryTableCreatorTest.java

```diff
@@ -392,6 +392,18 @@ public void fromBqResource_tableWithHyphen_throwsException() {
         "input (projects/test-project/datasets/audit_dataset/tables/lineage-table) not in correct format");
   }
 
+  @Test
+  public void fromBqResource_tableWithPartitionDecorator_valid() {
+    assertThat(
+        fromBigQueryResource(
+            "projects/example.co.in:test-project/datasets/audit_dataset/tables/lineage_table$20210606"))
+        .isEqualTo(
+            BigQueryTableEntity.create(
```

Comment:
May also add, Partition information in the table, so that different partitions can be distinguished.